actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -74,10 +82,12 @@ class ActionCable.Connection
 
   events:
     message: (event) ->
+      return if @disconnected
```

Comment:
Is there a scenario where messages are received after calling `webSocket.close()`?